main/display/wsg.c

```diff
@@ -323,6 +324,82 @@ void drawWsgSimple(const wsg_t* wsg, int16_t xOff, int16_t yOff)
     }
 }
 
+/**
+ * @brief Draw a WSG to the display without flipping or rotation
+ *
+ * @param wsg  The WSG to draw to the display
+ * @param xOff The x offset to draw the WSG at
+ * @param yOff The y offset to draw the WSG at
+ * @param xScale The amount to scale the image horizontally
+ * @param yScale The amount to scale the image vertically
+ */
+void drawWsgSimpleScaled(const wsg_t* wsg, int16_t xOff, int16_t yOff, int16_t xScale, int16_t yScale)
```

Comment:
Would be nice to combine rotation, reflection, and scaling one day. But not today.

Reply:
Ah, yeah I was going to just add a function with a transform matrix but then i realized it was a Bit More Complicated and just did this instead.